utils/datasets.py

```
@@ -0,0 +1,29 @@
+import os
+from typing import List
+
+import pandas as pd
+
+
+class Dataset:
+    def __init__(self, name: str, dataframe: pd.DataFrame):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)